src/main/java/com/hubspot/jinjava/lib/fn/Functions.java

```diff
@@ -145,4 +148,59 @@ public static int movePointerToJustBeforeLastWord(int pointer, String s) {
     return pointer + 1;
   }
 
+  @JinjavaDoc(value = "<p>Return a list containing an arithmetic progression of integers.</p>" +
+      "<p>With one parameter, range will return a list from 0 up to or down to the value. " +
+      " With two parameters, the range will start at the first value and end at the second value. " +
+      " The third parameter specifies the step increment.</p> <p>All values can be negative.</p>" +
+      "<p>Ranges can generate a maximum of " + RANGE_LIMIT + " values.</p>",
+      params = {
+          @JinjavaParam(value = "start", type = "number", defaultValue = "0"),
+          @JinjavaParam(value = "end", type = "number"),
+          @JinjavaParam(value = "step", type = "number", defaultValue = "1")})
+  public static List<Integer> range(Object arg1, Object... args) {
+
+    List<Integer> result = new ArrayList<>();
+
+    int start = 0;
+    int end;
+    int step = 1;
+
+    switch (args.length) {
+      case 0:
+        end = Integer.parseInt(arg1.toString());
```

Comment:
Is it worth wrapping this in a try/catch or otherwise doing some check to make sure the arguments are integers (to avoid throwing a `NumberFormatException`)?